src/Skedify.test.js

```diff
@@ -590,6 +590,32 @@ describe('API', () => {
       ).toMatchSnapshot()
     })
 
+    it('should be possible to disable certain actions (update) on a resource', () => {
+      expect(() =>
+        SDK.insights('cumul')
```

Comment:
Also not functional, but probably best to use `'cumulio'`, so new developers don't end up using the wrong type...